CONTRIBUTING.md

```diff
@@ -38,23 +38,10 @@ pip install -e .[docs,tests,extra]
 
 ## Codestyle
 
-We follow the [PEP8 codestyle](https://www.python.org/dev/peps/pep-0008/). Please order the imports as follows:
+We are using [black codestyle](https://github.com/psf/black) together with [isort](https://github.com/timothycrosley/isort)
+to sort the imports.
 
-1. built-in
-2. packages
-3. current module
-
-with one space between each,  that gives for instance:
-```python
-import os
-import warnings
-
-import numpy as np
-
-from stable_baselines3 import PPO
-```
-
-In general, we recommend using pycharm to format everything in an efficient way.
+**Please run `make format`** to reformat your code. You can check the codestyle using `make check-codestyle` and `make lint`.
```

Comment:
I would add here, or somewhere where it really stands out, that line-lengths are set to 127 rather than ~80. I think it is only included in the makefile now.